extensions-core/namespace-lookup/src/main/java/io/druid/server/lookup/namespace/cache/NamespaceExtractionCacheManager.java

```diff
@@ -153,15 +131,26 @@ public void run()
     );
   }
 
+  /**
+   * Optional monitoring for overriding classes. `super.monitor` does *NOT* need to be called by overriding methods
+   *
+   * @param serviceEmitter The emitter to emit to
+   */
+  protected void monitor(ServiceEmitter serviceEmitter)
```

Comment:
any reason we are doing the monitor within the cache manager and not in a separate monitor class?